pkg/templates/templates.go

```diff
@@ -824,6 +829,8 @@ func (t *TemplateResolver) processForAutoIndent(str string) string {
 		numSpaces := len(submatch[1]) - int(t.config.AdditionalIndentation)
 		matchStr := submatch[2]
 		newMatchStr := strings.Replace(matchStr, "autoindent", fmt.Sprintf("indent %d", numSpaces), 1)
+		newMatchStr = strings.Replace(newMatchStr, "toYaml", fmt.Sprintf("toYaml | indent %d", numSpaces), 1)
+		newMatchStr = strings.Replace(newMatchStr, "toYAML", fmt.Sprintf("toYAML | indent %d", numSpaces), 1)
```

Comment:
/hold I realized this replacement doesn't account for the `toYAML` argument--I'll have to adjust it to a RegEx replacement.

Reply:
This is resolved. Helm also notes that indentation processing can't be handled by the template processor and must be specified by the user. I'll create an issue to add hints/suggestions to error messages in this case.

https://github.com/helm/helm/blob/5f1bb7fee8de4d03f0651e317f9a513d02202d8d/pkg/engine/files.go#L99-L100

Reply:
New issue opened for error message hints: https://issues.redhat.com/browse/ACM-20011